pom.xml

```diff
@@ -201,6 +201,10 @@
     <version.net.byte-buddy>1.4.16</version.net.byte-buddy>
     <version.commons-beanutils>1.9.2</version.commons-beanutils>
     <version.docker-client>3.5.12</version.docker-client>
+    <version.gson>2.6.2</version.gson>
```

Comment:
Hello @errantepiphany , this gson 2.6.2 dependency is already defined in the  jboss-integration-platform-parent, so I believe we can just remove it form here.
I add a quick explanation since I don't know if you are familiar with our project structures.
In general all our projects like guvnor, kie-wb-commons and others has a something like this.

 jboss-integration-platform-parent <-  kie-parent <-  guvnor

if there is a dependency already defined in the jboss-integration-platform-parent we try to pick it, if we need to use a higher version number some times we override the version number in kie-parent, and then we try to see if this version number can be upgraded in jboss-integration-platform-parent by sending a PR to this project.